src/components/services/Providers.tsx

```diff
@@ -92,14 +53,6 @@ export function Providers(props: Props) {
       data: { context, disklet }
     })
 
-    // Watch the account:
```

Comment:
The subscription was only there to set component state for this component because it wasn't nested in the redux provider. The Services component is nested in the redux provider so it can get the state with a selector.